dist/js/main.js

```diff
@@ -0,0 +1,15 @@
+const burgerBtn = document.querySelector('.burger');
```

Comment:
logic of toggling isn't obvious. Could it be toggled by one element by one class?